CUSTOM.md

```diff
@@ -393,3 +393,11 @@ This section is meant for users who are familiar with both the tts mod and progr
       - `isGain`: **bool** - true if the operation is a gain, false if it's a pay
       - `isUndo`: **bool** - true if the operation is being undone (right click), false otherwise
       - `amount`: **number** - amount of energy gained/paid (before bargain deductions); positive for gain or pay
+- Trigger to modify the number of cards gained
+  - Create object and tag with "Modify Card Gain"
+  - `modifyCardGain(params)`
+    - `params`: **table** - contains data about the player and the cards played
+      - `color`: **string** - is the color of the player whose card cost is being calculated
+      - `minor`: **boolean** - true if gaining minors, false if gaining majors
```

Comment:
for power progression i believe `true` represents major so for consistency sake would probably be best to invert the boolean here

Reply:
even though the boolean before was whether it was minor or not, but now that it's public facing should probably be consistent

Reply:
I've changed it to `true` representing a major throughout the card gain code, for maximum consistency.